File: src/TouchOnline.Api/ClientApp/src/app/pages/management/lesson-text/shared/lesson-text.ts
export class LessonText {
    id:string;
    name: string;
    text: string;
    level: number;
    language: number;
    order: number;
    idLesson: number;
    
    static fromJson(jsonData: any): LessonText {
        let text = jsonData.text;
        let lesson = new LessonText();
        lesson.name = jsonData.name;
        lesson.level = parseInt(jsonData.level);
        lesson.language = parseInt(jsonData.language);
        text += this.getValue(jsonData.textLine2);
        text += this.getValue(jsonData.textLine3);
        text += this.getValue(jsonData.textLine4);
        text += this.getValue(jsonData.textLine5);
        text += this.getValue(jsonData.textLine6);
        text += this.getValue(jsonData.textLine7);
        lesson.text = text;
        lesson.order = jsonData.order;

        return lesson;
    }


    static getValue(value): string{
        if (value){
            return '¶' + value;
        } else {
            return '';
        }
    }
}

File: src/TouchOnline.Domain/LessonText.cs
using TouchOnline.Domain.Enums;

namespace TouchOnline.Domain
{
    public class LessonText : Entity
    {
        public int IdLesson { get; set; }
        public string Name { get; set; }
        public string Text { get; set; }
        public Level Level { get; set; }
        public Language Language { get; set; }
        public int Order { get; set; }
        public User User { get; set; }
    }
}
